### ISSUE TITLE
Staging Review finding: "Finish this application later" cannot be accessed by keyboard

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#115280]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Lifestage Pension and Burial
**Product:** Pension
**Feature:** Intent to File V1

## Findings details
**VA.gov Experience Standard - issue:** [User can perceive all meaningful elements.](https://design.va.gov/about/experience-standards/#user-can-perceive-all-meaningful-elements)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @rsmithadhoc (Accessibility)

### Description

When tabbing through the interactive elements on the page, it skips the "Finish this application later" link.

<details><summary>Link, screenshot or steps to recreate</summary>

Any page throughout the form steps.

![Image](https://github.com/user-attachments/assets/1f917a51-a015-48af-a0f9-e93841a1cd83)

</details>

## Recommended action

I'm not sure of the exact cause, but ensure that there is no manual tabindex set on the element and that the visual order of the elements on the page matches the order in the DOM. 

## Acceptance Criteria

- [x] "Finish this application later" link is keyboard accessible
- [x] Unit tests in place to prevent regression
- [x] All tests passing

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 2.1.1 Keyboard - Level A](https://www.w3.org/TR/WCAG22/#keyboard)
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by matt4su** [comment]
[Spike: Identify Other Forms with same issues as 0969/ITF launch blockers](https://github.com/department-of-veterans-affairs/va.gov-team/issues/115164)

**Comment by aplatt-coforma** [comment]
@skylerschain - can you please help QA this ticket today? Thanks!

**Comment by skylerschain** [comment]
Design QA ✅
